marketplace-service/src/main/java/ch/akros/marketplace/service/controller/TopicController.java

```diff
@@ -29,13 +31,26 @@ public ResponseEntity<TopicLoadResponseDTO> topicsTopicIdGet(Long topicId) {
         }
     }
 
+    @GetMapping(value = "/topics/user/{userId}")
+    public ResponseEntity<List<TopicLoadResponseDTO>> getAllTopicsFromUser(@PathVariable("userId") String userId) {

```

Comment:
I think, logically woul be here to name the method getTopicsForUser.

Reply:
Why ..forUser? In german that would be "Themen für Benutzer", but the topics are not for him :D, They are from him :)

Reply:
Logically you are right. 
But here we are talking about the name of the method, which returns the topics according to the specific User. It means, return Topics for given user. And if I use the preposition "from" here, for it means: From the list of Users, I select Topics.

Reply:
oke I have adapt it.